hack/force-cert-retation.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -xe
+
+KUBECTL=${KUBECTL:-kubectl}
+POD_NAMESPACE=${POD_NAMESPACE:-kubemacpool-system}
+POD_APP_LABEL=${POD_APP_LABEL:-kubemacpool-leader=true}
+WEBHOOK_TYPE=${WEBHOOK_TYPE:-mutatingWebhookConfiguration}
+WEBHOOK_NAME=${WEBHOOK_NAME:-kubemacpool-mutator}
+TEMP_FOLDER=${TEMP_FOLDER:-_tmp}
+
+WEBHOOK_CONFIG_YAML_DIR=$(pwd)/${TEMP_FOLDER}/
+WEBHOOK_YAML_FILE=${WEBHOOK_CONFIG_YAML_DIR}/${WEBHOOK_TYPE}_${WEBHOOK_NAME}.yaml
+
+mkdir -p ${WEBHOOK_CONFIG_YAML_DIR}
+
+# get webhook config in yaml format
+${KUBECTL} get ${WEBHOOK_TYPE} -n ${POD_NAMESPACE} ${WEBHOOK_NAME} -oyaml > ${WEBHOOK_YAML_FILE}
+
+#remove the caBundle from the webhook config
+sed '/caBundle:/d' ${WEBHOOK_YAML_FILE} > ${WEBHOOK_YAML_FILE}.out
```

Comment:
This will not work when caBundle is set as a multiline string

Reply:
I'm not sure what you mean. isn't the caBundle a very long string by default?

Reply:
Both of the following are valid yaml strings represented as a single line:

```
foo: very long line
```

```
foo: "very
long
line"
```

```
foo: >
  very
  long
  line
```

Not sure they will be usefull for BASE64, since they include spaces between words though. But the point is, using singleline sed replacement may not always work. But if we always create a single line in this library, then this all is irrelevant :)

Reply:
I looked at the pem format RFC. couldn't find a line like "PEM format is one line string" - but I'm fairly sure it is. @qinqon - do you think sed is the way to go ?